layers/openflow.go

```diff
@@ -0,0 +1,718 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+// Copyright 2014-2015 Vasiliy Tolstov. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/google/gopacket"
+)
+
+var LayerTypeOpenflow = gopacket.RegisterLayerType(12340, gopacket.LayerTypeMetadata{"Openflow", gopacket.DecodeFunc(decodeOFP)})
+
+const (
+	OFPMC_ADD    = 0 /* New meter. */
```

Comment:
To match with other enums in gopacket, please use CamelCase instead of CAPS_WITH_UNDERS, here and below.

Probably prefix all with "Openflow" or "OpenFlow".